examples/fishnet_api/main.py

```diff
@@ -296,7 +347,7 @@ async def upload_algorithm(algorithm: UploadAlgorithmRequest) -> Algorithm:
     """
     if algorithm.id_hash is not None:
         # update existing algorithm
-        resp = await Algorithm.fetch(algorithm.id_hash)
+        resp = await Algorithm.fetch(algorithm.id_hash).all()
         old_algorithm = resp[0] if resp else None
```

Comment:
can be replaced with `.first()`
```suggestion
        old_algorithm = await Algorithm.fetch(algorithm.id_hash).first()
```